src/components/Table/Table.tsx

```diff
@@ -17,18 +21,39 @@ export const Table = ({
   children,
   fullWidth,
   fixed,
+  scrollable,
+  striped,
+  compact,
+  stackedStyle = 'none',
 }: TableProps): React.ReactElement => {
   const classes = classnames('usa-table', {
     'usa-table--borderless': !bordered,
     [`${styles.fullwidth}`]: fullWidth,
     [`${styles.fixed}`]: fixed,
+    'usa-table--striped': striped,
+    'usa-table--compact': compact && stackedStyle === 'none',
+    'usa-table--stacked': stackedStyle === 'default',
+    'usa-table--stacked-header': stackedStyle === 'headers',
   })
 
   return (
-    <table className={classes} data-testid="table">
-      {caption && <caption>{caption}</caption>}
-      {children}
-    </table>
+    <>
+      {scrollable ? (
+        <div
+          data-testid="scrollable-table"
+          className="usa-table-container--scrollable">
+          <table className={classes} data-testid="table">
+            {caption && <caption>{caption}</caption>}
+            {children}
+          </table>
+        </div>
+      ) : (
+        <table className={classes}>
+          {caption && <caption>{caption}</caption>}
+          {children}
+        </table>
+      )}
+    </>
```

Comment:
If you extract out table into a const, then you won't need to repeat it in your ternary:

```TSX
const table = (
  <table className={classes} data-testid="table">
    {caption && <caption>{caption}</caption>}
    {children}
  </table>
)
```

```suggestion
    <>
      {scrollable ? (
        <div
          data-testid="scrollable-table"
          className="usa-table-container--scrollable">
          {table}
        </div>
      ) : (
        {table}
      )}
    </>
```